_features/auto_bed_leveling.md

```diff
@@ -47,4 +47,8 @@ Begin Automatic Bed Leveling with a plain `G29` command. This will use the setti
 If all goes well, move the nozzle down close to the bed and use a piece of paper to test the nozzle height. The paper should slide with a little bit of catch, but not too much. Now move the nozzle to different points on the bed with `G1` and re-test the height with the paper at each point. The feel should be close to the same at all points. If you find that the leveling isn't very accurate, use `M48` to test the accuracy of the bed probe.
 
 ## Bed Leveling and Printing
-Currently, Marlin doesn't save the bed leveling results to EEPROM, and it throws away the probe results on `G28`. So the machine must always do a new `G29` after any `G28`. Marlin will be getting a Save Leveling function in the near future, and we'll make an announcement at that time.
+Marlin saves the bed leveling results to EEPROM after each `G29` command. It will remain in memory until the next `G29` or homing by `G28`. After each `G28` a simple `M420 S1` will restore the saved mesh to memory. It is advised to set your slicer to add the `M420 S1` in the starting code.
+
+In order to load the mesh from the printer boot up, after the `M420 S1` you can save this ABL active satus with `M500`. 
```

Comment:
The active status of leveling is not saved and restored for ABL. Leveling is left off until the user explicitly enables it with `M420 S1` or a new `G29`. On bootup the printer needs a `G28` before it can do anything else, and this command (naturally) disables leveling as part of its specification. The appropriate behavior is to leave the printer in literal-move mode until you start a print job, and users may also enable leveling to perform further calibration.

Another problem with having leveling on before you print is that moves performed from the LCD with "Move Axis" are not leveled for all kinematics, so during setup the Z offset could get screwed up without the user being aware.